clients/src/main/java/org/apache/kafka/clients/producer/ProducerRecord.java

```diff
@@ -12,6 +12,8 @@
  */
 package org.apache.kafka.clients.producer;
 
+import org.apache.kafka.common.record.Record;
+
 /**
  * A key/value pair to be sent to Kafka. This consists of a topic name to which the record is being sent, an optional
  * partition number, and an optional key and value.
```

Comment:
Since we are adding timestamp field to ProducerRecord, I think we should add a comment to ProducerRecord class description about meaning of timestamp, what happens if user sets null, etc.